src/main/scala/org/scalacheck/Cogen.scala

```diff
@@ -30,8 +30,7 @@ sealed trait Cogen[T] extends Serializable {
 
 object Cogen extends CogenArities with CogenLowPriority {
 
-  // See https://github.com/rickynils/scalacheck/issues/230 for dummy expl.
-  def apply[T](implicit ev: Cogen[T], dummy: Cogen[T]): Cogen[T] = ev
```

Comment:
this broke bincompat

Reply:
Yes it did.  I believe it should have only been released in 1.14.0, and not in 1.13.x.

Reply:
hmm. Interesting. I am seeing the breakage in 1.13.x.

Reply:
it is possible something is transitively pulling in 1.14.x in my project. I'll check that.